### PR TITLE
Issue #7329 fixed

### PR DESCRIPTION
**Pull request by suraj-webkul**
Exchange rate api updated

### CODE REVIEW

File: packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function  __construct(
         protected ExchangeRateRepository $exchangeRateRepository
     )
     {
-        $this->apiEndPoint = 'https://api.exchangeratesapi.io/latest';
+        $this->apiEndPoint = 'https://api.apilayer.com/exchangerates_data/convert';
```
Comment (devansh-webkul) [inline packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php:37]: Add URL in services.php file.

~~~php
config('services.exchange_api.exchange_rates.url')
~~~

Comment (devansh-webkul) [reply packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php:37]: And also make the `exchange-api` key to `exchange_api`.

File: packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php [hunk 2]
```diff
@@ -53,7 +53,15 @@ public function updateRates()
                 continue;
             }
 
-            $result = $client->request('GET', $this->apiEndPoint . '?access_key='. $this->apiKey . '&base=' . config('app.currency') . '&symbols=' . $currency->code);
+            $result = $client->request(
+                'GET',
+                $this->apiEndPoint . '?to=' . $currency->code . '&from=' . config('app.currency') . '&amount=1', [
```
Comment (devansh-webkul) [inline packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php:58]: I think there must be an argument from where you can add params. Just check that and remove the `?` dependency.

File: packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php [hunk 3]
```diff
@@ -66,11 +74,11 @@ public function updateRates()
...
-                    'rate' => $result['rates'][$currency->code],
+                    'rate' => $result['result'],
                 ], $exchangeRate->id);
             } else {
                 $this->exchangeRateRepository->create([
-                    'rate'            => $result['rates'][$currency->code],
+                    'rate'            => $result['result'],
                     'target_currency' => $currency->id,
                 ]);
             }
```
Comment (devansh-webkul) [inline packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php:84]: Add a switch case here and check for both two and if the third one or invalid comes up throw an exception (like InvalidExchangeRateException). No need of handling the invalid part.

Comment (devansh-webkul) [reply packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php:84]: You can use if cases also. But else part should throw an exception.

Comment (devansh-webkul) [reply packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php:84]: Sorry, ignore the wrong area I added.

File: packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php (hunk 1 above)
Comment (suraj-webkul) [reply packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php]: Done

File: packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php (hunk 2 above)
Comment (suraj-webkul) [reply packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php]: Done

**Comment by Sanjaybhattwebkul** [comment]
@devansh-webkul 
Please review this PR.

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**